test/containers/comment-decorations-container.test.js

```diff
@@ -184,8 +184,10 @@ describe('CommentDecorationsContainer', function() {
 
     it('renders nothing if query errors', function() {
       const tokenWrapper = localRepoWrapper.find(ObserveModel).renderProp('children')('1234');
+      const stub = sinon.stub(console, 'warn');
       const resultWrapper = tokenWrapper.find(QueryRenderer).renderProp('render')({
         error: 'oh noes', props: null, retry: () => {}});
+      stub.restore();
```

Comment:
This `.restore()` call isn't strictly necessary - we set up `sinon` as a sandbox that automatically gets restored after each test. 🌠

Reply:
good to know. I am going to properly stub or fix these in a new PR as right now they consume too much.